### ISSUE TITLE
"toHuman" uses Intl.ListFormat, which isn't supported by Safari < 14.1

### ISSUE THREAD

**Issue by distinctdan** [moment/luxon#1189]
**Describe the bug**
I have a user who's on an older version of Safari, and they're getting an error about Intl.ListFormat. Looking through the luxon source, it looks like it's coming from `toHuman`. Unfortunately, `Intl.ListFormat` isn't supported by older versions of safari. Here's the error message (manually typed from a screenshot):

```
TypeError: undefined is not a constructor (evaluating "new Intl.ListFormat(e, t)")
```

It looks like this was added to luxon fairly recently, here's the place in the source:
https://github.com/moment/luxon/blob/c96b8132661ae610b22f2318d1cfde5f262fd5d1/src/impl/locale.js#L14

**To Reproduce**
Please share a minimal code example that triggers the problem:
I haven't been able to get a stack trace from our user, but here's where I think it's coming from:
```javascript
dur.shiftTo('days').toHuman({
    unitDisplay: 'short',
    maximumFractionDigits: 1,
});
```

**Desktop (please complete the following information):**

- OS: [OSX]
- Browser [Safari, unknown version, sorry]
- Luxon version [2.3.0]
- Your timezone [America/New_York]

**Additional context**
CanIUse: https://caniuse.com/?search=intl.listFormat
This is also referenced here by this feature discussion: https://github.com/moment/luxon/issues/688#issuecomment-1024634001

**Comment by icambron** [comment]
Yeah, I'd accept a fix that feature detects and just falls back to just using commas

**Comment by shawnlauzon** [comment]
I'm hitting an error while using `toHuman` on React Native also, looks like the same issue.

**Comment by JavanPoirier** [comment]
In the meantime you can just use formatjs polyfills, I have it working with babel/expo.
https://formatjs.io/docs/polyfills/intl-listformat

**Comment by codewizardVN** [comment]
> I'm hitting an error while using `toHuman` on React Native also, looks like the same issue.

Did you solve the problem, I think you can use ```formatjs``` to solve it!

Install module: 
> yarn add @formatjs/intl-listformat @formatjs/intl-getcanonicallocales @formatjs/intl-locale

Import modules inside ```index.js```
> import '@formatjs/intl-getcanonicallocales/polyfill';
   import '@formatjs/intl-locale/polyfill';
   import '@formatjs/intl-listformat/polyfill';
   import '@formatjs/intl-listformat/locale-data/en';

Good luck!
